website/static/templates.json

```diff
@@ -5826,31 +5826,54 @@
     ]
   },
   {
-  "title": "Copilot Studio with Azure AI Search using Azure Developer CLI",
-  "description": "An enterprise-ready template demonstrating secure integration between Microsoft Copilot Studio and Azure AI Search. Features private networking, Power Platform integration, and comprehensive AI search capabilities with enterprise-grade security and governance. Deployed using Azure Developer CLI (azd) with Terraform infrastructure-as-code.",
-  "preview": "./templates/images/copilot-studio-azure-ai-search.png",
-  "authorUrl": "https://github.com/Azure-Samples",
-  "author": "ISE Scale",
-  "source": "https://github.com/Azure-Samples/Copilot-Studio-with-Azure-AI-Search",
-  "tags": [
-    "terraform",
-    "powershell",
-    "msft",
-    "ai",
-    "new"
-  ],
-  "azureServices": [
-    "aisearch",
-    "azureai",
-    "azurestorage",
-    "vnets",
-    "privateEndpoints",
-    "managedidentity",
-    "appinsights"
-  ],
-  "id": "b6399e1c-612b-4d43-8eda-ccf226213fe6",
-  "languages": [
-    "python"
-  ]
-}
+    "title": "Copilot Studio with Azure AI Search using Azure Developer CLI",
+    "description": "An enterprise-ready template demonstrating secure integration between Microsoft Copilot Studio and Azure AI Search. Features private networking, Power Platform integration, and comprehensive AI search capabilities with enterprise-grade security and governance. Deployed using Azure Developer CLI (azd) with Terraform infrastructure-as-code.",
+    "preview": "./templates/images/copilot-studio-azure-ai-search.png",
+    "authorUrl": "https://github.com/Azure-Samples",
+    "author": "ISE Scale",
+    "source": "https://github.com/Azure-Samples/Copilot-Studio-with-Azure-AI-Search",
+    "tags": [
+      "terraform",
+      "powershell",
+      "msft",
+      "ai",
+      "new"
+    ],
+    "azureServices": [
+      "aisearch",
+      "azureai",
+      "azurestorage",
+      "vnets",
+      "privateEndpoints",
+      "managedidentity",
+      "appinsights"
+    ],
+    "id": "b6399e1c-612b-4d43-8eda-ccf226213fe6",
+    "languages": [
+      "python"
+    ]
+  },
+  {
+    "title": "Remote self-hosted Azure MCP Server with managed identity for Copilot Studio integration",
+    "description": "This template demonstrates how to self-host a remote Azure MCP Server as an Azure Container App which can use its managed identity to read Azure resources. This template also creates Entra app registrations for connecting Copilot Studio to this remote Azure MCP Server.",
+    "preview": "./templates/images/azmcp-copilot-studio-aca-mi.png",
+    "authorUrl": "https://github.com/microsoft/mcp/tree/main/servers/Azure.Mcp.Server",
+    "author": "Chunan Ye",
+    "source": "https://github.com/Azure-Samples/azmcp-copilot-studio-aca-mi",
+    "tags": [
+      "bicep",
+      "mcp",
+      "msft",
+      "ai",
+      "new"
+    ],
+    "azureServices": [
+      "aca",
+      "managedidentity"
```

Comment:
We'd want to add appInsights here. What about storage managed identity since its used in the sample?

Reply:
Managed identity is already covered here. I didn't include app insights because it's disabled by default in the template. If someone clones the template and run `azd up` there will be no app insights in the provisioned resources.

Reply:
There is no particular reason. I kept it disabled when I adapated Anu's template for foundry. It's a good point that the implementation doesn't emit useful telemetry so there is little point in enabling it right now. We can definitely enable it by default and add instructions on how to look at the telemetry once azmcp emits more useful telemetry data.